Program.cs

```diff
@@ -3,14 +3,17 @@
 using System.Windows.Input;
 using System.Windows.Controls;
 using System.Windows.Media.Imaging;
+using static System.Math;
+using static System.Runtime.InteropServices.JavaScript.JSType;
 
 namespace A25;
 
 class Point2D {
    public readonly int X;
    public readonly int Y;
-   public Point2D (double x, double y) => (X, Y) = ((int)x, (int)y);
-   public static double Distance (Point2D p1, Point2D p2) => Math.Sqrt (Math.Pow (p2.X - p1.X, 2) + Math.Pow (p2.Y - p1.Y, 2));
+   public Point2D (double x, double y) => (X, Y) = ((int)(x + 0.5), (int)(y + 0.5));
+   public static double Distance (Point2D p1, Point2D p2) => Sqrt (Pow (p2.X - p1.X, 2) + Pow (p2.Y - p1.Y, 2));

```

Comment:
The sqrt computation is a costly operation. This computation can be done in the constructor itself and can be used whenever required.

Reply:
I have removed this Distance method in the b8b8175 commit since its not used anywhere